client/src/components/Home/index.tsx

```diff
@@ -314,13 +314,27 @@ export default class Home extends React.Component<any, IHomeState> {
     );
 
     // Dividing templates into categories
-    let categories = { 'General': [] };
+    let categories = { 'General': [], 'Featured': [] };
     templates.forEach((tmpl, index) => {
       let category = tmpl.category || 'General';
+      if (tmpl.featured) {
```

Comment:
Since this is a separate logical part, I'd put a space line before and after the `if { /*...*/ }`